core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -215,6 +215,12 @@ oppia.controller('CollectionPlayer', [
       return completedExplorationIds.indexOf(explorationId) > -1;
     };
 
+    $scope.isNextExploration = function(explorationId) {
```

Comment:
Can we just do this once on page load, instead of having a function that's called once for every node on the page? Store the "next" exploration in a variable and refer to that instead.